Data/operators.csv

```diff
@@ -24,7 +24,9 @@ map,"map, tryMap",
 merge,"merge, tryMerge",
 multicast,multicast,
 observeOn,receive(on:),
+publish,makeConnectable

```

Comment:
Can you also make . sure you add the last comma so the last column is empty? (It's optional but I prefer keeping it standard)

Reply:
Sure, done